pkgs/development/libraries/libpfm/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
 
   installFlags = "DESTDIR=\${out} PREFIX= LDCONFIG=true";
 
+  makeFlags = [
+    "ARCH=${stdenv.targetPlatform.uname.processor}"
```

Comment:
Just a note - libpfm looks to be using ```uname -m```. This is technically supposed to be ```uname -p```. Are those supposed to be identical?

Reply:
Very weird! Is this doc incorrect:

https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_PROCESSOR.html#variable:CMAKE_HOST_SYSTEM_PROCESSOR

? At least cmake should use uname -m in this case.

Reply:
It does: https://fuchsia.googlesource.com/third_party/cmake/+/v3.5.1/Modules/CMakeDetermineSystem.cmake#50